### DISCUSSION TITLE
Large discrepancy between GFN methods for large systems

### DISCUSSION THREAD

**Discussion by andrewtarzia** [grimme-lab/xtb#548] [Q&A] [answered]
Hello!

I am unsure if this is an issue or a discussion point, but I would really appreciate some checks on my approach and ideas for the causes of these large changes.

I have two metal-organic cage systems of 668 and 1336 atoms and +16 and +32 charge.

I optimised the structures (starting from crystal structures) with GFN2 (using version 6.4.0 (d4b70c2) of xtb throughout) in the gas phase and found that I needed implicit solvation to get any reasonable energy comparisons. However, the optimisation in implicit solvation with GFN2 is slow - so, I tested GFN-FF, GFN1 and GFN0.

My main issue is that when I use GFN0 or GFN1, I get explosions of the systems (see attached XYZ files of the start and end of a GFN0 optimisation with alpb==water; same result in gas phase). Note that GFN-FF runs fine and that these are all initiated from a GFN2 optimised structure.

A command-line run looks like this: 

`ulimit -s unlimited ; nohup /home/atarzia/software/xtb-6.4.0/bin/xtb jd362_xtb.xyz --gfn 0 --opt tight --chrg 32 -P 6 --alpb water > jd362_xtb_gfn0_water.out &`

Am I using xtb incorrectly? Or missing some key information about the difference between the methods for high charge species? How can I best diagnose this issue?

[for_discussion_0.txt](https://github.com/grimme-lab/xtb/files/7698409/for_discussion_0.txt)
[for_discussion_5382.txt](https://github.com/grimme-lab/xtb/files/7698410/for_discussion_5382.txt)

**Comment by haneug** [comment, marked as answer]
It makes sense that for large highly charged metal-organic cage systems the SCC convergence becomes difficult and that an implicit solvation model, which screens the charges, helps with convergence. Without the solvation model charges become too delocalized which can lead to artificial charge distributions (which can lead to explosions/fusion). Nevertheless, even with an implicit solvation model converging these cages is a difficult task, which might not always be doable.

Regarding the different GFN methods:
1. GFN-FF is a force field and therefore no SCC is done at all.
2. GFN1-xTB is similar to GFN2-xTB but without multipole expansion.  
3. GFN0 is a non-iterative tight-binding method, where the electrostatics are determined via a classical electronegativity equilibration (EEQ) model. This model likely has problems with such large charges. Also, the implicit solvation models are not optimized for GFN0 (I believe the GFN2-xTB parameters are taken for solvation here).

The question is what GFN method fits best to your problem.
If you want to use SQM methods, I recommend using GFN2-xTB with an implicit solvation model (it is faster than GFN1-xTB, because the basis set is a little bit smaller for H). If GFN2-xTB is too slow for your application, I recommend GFN-FF if the geometries are satisfactory (but note that no bonds can be created with GFN-FF, only broken). I would use GFN0-xTB only if it can sufficiently describe my problem, which seems to be not the case for these cages.

**Comment by andrewtarzia** [comment]
Thank you for this, that is very helpful!

Are there any recommendations of settings to try/use for these systems for GFN2? So far, the defaults have always been reliable for me, so I tend to stick to them!

**Reply by haneug** [reply]
You can stick to the defaults and if you run into convergence problems you can try the sad guess (superposition of atomic densities), which is often more stable. To do so generate an xtb.inp file with '--copy --define' and add to the $scc block:
`guess=sad `
then read the xtb.inp file via '--input'.
Another tool for convergence is the Fermi smearing. 
First converge the SCC with a high electronic temperature e.g. 
`xtb mol.xyz --etemp 5000`
and then restart on this solution with the default temperature of 300 K.
I hope this helps!

**Reply by andrewtarzia** [reply]
Thank you!
